data-migration/2.x-to-3.x/harvest_project_info_nlesc.py

```diff
@@ -670,7 +670,8 @@ def update_project_in_database(
         return True
     else:
         logger.warning(
```

Comment:
Nice, did not get any failed projects during my run, but good to see some more feedback on failures.